db/seeds/sandbox_data.rb

```diff
@@ -0,0 +1,17 @@
+require 'faker'
+
+
+Faker::Config.locale = 'en'
+
+providers = ['Capita', 'Teach First', 'UCL', 'Best Practice Network', 'Ambition', 'Education Development Trust']
+providers.each do |provider_name|
+  lead_provider = LeadProvider.find_or_create_by!(name: provider_name)
+  token = LeadProviderApiToken.create_with_random_token!(lead_provider: lead_provider)
```

Comment:
Are you sure you want random token each time you seed it? It can be quite a faff to replace them :/ 
Although I guess you aren't planning to drop it any time soon, so it's probably alright.

Reply:
Agree. I think the idea was to publish the tokens or a token on the sandbox landing page so providers could make test requests. We either have one dummy provider with a token or a list of providers

Reply:
Sandbox tokens will be sent by email.